keyboards/plain60/keymaps/RGB/keymap.c

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Sebastian Spindler <sebastian.spindler@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#include "keymap_extras/keymap_german.h"
+
+//Layer renaming
+#define _DL  0                     //default
+#define _FUN 1                     //function layer
+#define _LED 2                     //LED configurations
+
+//Keymapping renaming
+#define ______  KC_TRNS            //renaming KC_TRNS for readability in keymaps
```

Comment:
If you remove this define and use the QMK-supplied one (which is 7 characters long), you can align your keymap much better.